tests/db/utils_test.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import absolute_import
 
 import email
+import os
+import os.path
```

Comment:
I think this is a special case for `os.path` (maybe `os` imports it by default) so I would not like to depend on it (unless it is documented?). In general submodules are not automatically available:
```python
>>> import email
>>> email.header
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'header'
>>> import email.header
>>> email.header
<module 'email.header' from '/usr/lib/python2.7/email/header.pyc'>
>>> import os
>>> os.path
<module 'posixpath' from '/usr/lib/python2.7/posixpath.pyc'>
>>> 
```